plugins/artifacts/artifacts_test.go

```diff
@@ -0,0 +1,136 @@
+package artifacts
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/taskcluster/slugid-go/slugid"
+	"github.com/taskcluster/taskcluster-client-go/queue"
+	"github.com/taskcluster/taskcluster-client-go/tcclient"
+	"github.com/taskcluster/taskcluster-worker/plugins/plugintest"
+	"github.com/taskcluster/taskcluster-worker/runtime/client"
+)
+
+type artifactTestCase struct {
+	TestCase  *plugintest.Case
```

Comment:
This is a neat trick!!! :)

You can probably just embed `plugintest.Case` removing some unneeded intentation...

You can still call Test() by using `a.plugintest.Case.Test()` or `a.Case.Test()` I think...